content/applications/productivity/knowledge/properties.rst

```diff
@@ -0,0 +1,81 @@
+==========
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with **write**
+access. These fields are shared between all the child articles and article items under the same
+parent.
+
+.. note::
+   To be able to add properties, an article must be either a **child article** or an **article
+   item**.
```

Comment:
I assume you'll add custom links once the pages describing this are created

Reply:
I confirm